blog/_posts/2017-02-22-repl-0.6-highlights.md

```diff
@@ -0,0 +1,108 @@
+---
+layout: post
+title:  "Upgrades to the REPL in v0.6"
+author: <a href="https://kristofferc.github.io/">Kristoffer Carlsson</a>
+---
+
+Since v0.3, Julia comes with a REPL completely written in Julia ([PR #6270](https://github.com/JuliaLang/julia/pull/6270)).
+Among other features, the REPL provides code completion, history with reverse search, a rich set of keybindings which in addition are customizable. (See the [Interacting with Julia](http://docs.julialang.org/en/stable/manual/interacting-with-julia/) part of the manual for the REPL documentation.)
+Even so, v0.6 of Julia comes with quite a few new visual tweaks and feature upgrades over v0.5.
```

Comment:
Even so, the REPL in Julia 0.6 includes quite a few new visual tweaks and feature upgrades.

("v" is not really a standard English abbreviation for "version".)